cdm/src/main/java/ucar/nc2/dataset/conv/CF1Convention.java

```diff
@@ -135,7 +135,19 @@ public void augmentDataset(NetcdfDataset ds, CancelTask cancelTask) throws IOExc
           v.addAttribute(new Attribute(_Coordinate.AxisType, AxisType.Ensemble.toString()));
           continue;
         }
-
+        
+        //sjk code

```

Comment:
What is sjk?

Reply:
It was a placeholder I accidentally left in. I've already removed it.